src/index.css

```diff
@@ -1,19 +1,24 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+
 * {
```

Comment:
- [ ] kindly consider fixing your design by sticking to the given one as much as possible  to make your design look exactly as expected check on the image below to see the main parts I highlighted
<img width="1420" alt="Screen Shot 2023-06-22 at 18 19 49" src="https://github.com/Nahid1911/bookstore_using_react/assets/60146030/90e97e2b-e6c7-41f8-8476-2fc92b37daf4">

Reply:
Hello @divinecharlotte 
Thank you for your review.
1. Font Family and small gray line resolved.
2. Border around button removed.
3. Font width is not mention in the Zeplin design.
4. Author to Categories, I would like to mention that in the first day project requirements mentioned as follows, 
<img width="530" alt="image" src="https://github.com/Nahid1911/bookstore_using_react/assets/118947090/e8bc829c-4fbc-4c86-9f7e-84a9133205a6">

please visit the link of [first day project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md)